spec/index.bs

```diff
@@ -5136,10 +5136,11 @@ enum GPUMipmapFilterMode {
 </dl>
 
 {{GPUCompareFunction}} specifies the behavior of a comparison sampler. If a comparison sampler is
-used in a shader, an input value is compared to the sampled texture value, and the result of this
-comparison test (0.0f for pass, or 1.0f for fail) is used in the filtering operation.
+used in a shader, an input value is compared to each sampled texel value, and the result of this
+comparison test is generated (`0.0f` for pass, or `1.0f` for fail).
 
-Issue: describe how filtering interacts with comparison sampling.
+The comparison results for each texel may be blended together with the sampler's normal
+texture filtering, with the resulting value in the range `[0..1]` returned.
```

Comment:
WDYT:
```suggestion
If texture filtering is enabled, the filtering step occurs after comparison, so that comparison
results are mixed together resulting in values in the range `[0, 1]`. Filtering **should** behave
as usual, however it **may** be computed with lower precision or not mix results at all.
```

Reply:
I like that better, thanks. Feels like it more completely describes the expected range of behavior.